main.cpp

```diff
@@ -231,214 +333,320 @@ struct BasketballCourt
         std::string appForOrganizing = "Notion";
 
         void cancelAllMondayEvents(int numberOfMondayEvents);
-        void rescheduleAllWendesdayEvents(int numberOfWendesdayEvents);
+        void rescheduleAllWednesdayEvents(int numberOfWednesdayEvents);
         void organizeNextMonthsSchedule();
     };
 
-    //3 things it can do:
-    //    1) host basketball games
     void hostBasketballGames(EventsSchedule scheduledGameDay);
-    //    2) host sport events
     void hostSportEvents();
-    //    3) sell tickets
-    float sellTickets(float ticketPrice); //returns the amount of money earned
+    float sellTickets(float ticketPrice);
+
     void scheduleNewGameDay(EventsSchedule newGameDay);
     EventsSchedule scheduledGameDay;
 };
 
-/*
-Thing 4) cafeteria
+void BasketballCourt::EventsSchedule::cancelAllMondayEvents(int numberOfMondayEvents)
+{
+    std::cout << numberOfMondayEvents << "events have been cancelled";
+}
+
+void BasketballCourt::EventsSchedule::rescheduleAllWednesdayEvents(int numberOfWednesdayEvents)
+{
+    std::cout << numberOfWednesdayEvents << "events have been rescheduled";
+}
+
+void BasketballCourt::EventsSchedule::organizeNextMonthsSchedule()
+{
+    int numberOfNextMonthsEvents = 18;
+    std::cout << "organize" << numberOfNextMonthsEvents << "next month";
+}
+
+void BasketballCourt::hostBasketballGames(EventsSchedule gameDay)
+{
+    gameDay.rescheduleAllWednesdayEvents(2);
+}
+
+void BasketballCourt::hostSportEvents()
+{
+    std::string nameOfEvent = "event name";
+    std::string eventDate = "date";
+    std::cout << nameOfEvent << "will take place on" << eventDate;
+}
+
+float BasketballCourt::sellTickets(float ticketPrice)
+{
+    int numberOfTicketsSold = 651.0;
+    float ticketsFloat;
+    ticketsFloat = float(numberOfTicketsSold);
```

Comment:
Casting has not been covered, and this is a C-Style cast.
remove it
find another way to solve the warnings related to this type conversion.
- [ ] revise